acme/acme/magic_typing.py

```diff
@@ -0,0 +1,16 @@
+"""Shim class to not have to depend on typing module in prod."""
+import sys
+
+class TypingClass(object):
+    """Ignore import errors by getting anything"""
+    def __getattr__(self, name):
+        return None
+
+try:
+    # mypy doesn't respect modifying sys.modules
+    from typing import * # pylint: disable=wildcard-import, unused-wildcard-import
```

Comment:
I think we can just have this `from typing import *` here and remove the `sys.modules` stuff from the `try` block. We still need it in the `except` case though.

Reply:
it cleans it up a bit by sticking the thing directly in there, but it might not be necessary -- I had this first before adding `from typing import *` so I don't actually remember if it's necessary. it might be though, this whole process is a bit testy.